src/transformers/image_processing_utils.py

```diff
@@ -13,13 +13,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from .feature_extraction_utils import BatchFeature as BaseBatchFeature
 from .feature_extraction_utils import FeatureExtractionMixin
 from .utils import logging
 
 
 logger = logging.get_logger(__name__)
 
 
+# TODO: Move BatchFeature to be imported by both feature_extraction_utils and image_processing_utils
+# We override the class string here, but logic is the same.
+class BatchFeature(BaseBatchFeature):
+    r"""
+    Holds the output of the image processor specific `__call__` methods.
+
+    This class is derived from a python dictionary and can be used as a dictionary.
+
+    Args:
+        data (`dict`):
+            Dictionary of lists/arrays/tensors returned by the __call__/pad methods ('pixel_values', 'attention_mask',
+            etc.).
+        tensor_type (`Union[None, str, TensorType]`, *optional*):
```

Comment:
A quick question - are we going to default to "np" ? If so, maybe we can remove None from the accepted argument types or make it a non-optional argument

Reply:
It depends partly on whether we merge in: https://github.com/huggingface/transformers/pull/18499

Defaulting to `"np"` isn't necessary to be able to use different combinations of e.g. `do_resize` and `do_normalize`. As we're aliasing the previous feature extractors with the new image processors, change the default would still be a breaking change. 

If we decided to default to `"np"`, then we'd have to include additional checks on the processed images. At the moment, because `resize` resizes the images to multiples of `size_divisor`, they are not guaranteed to all be the same size. This means calls the `BatchFeature` will fail if any of `"np"`, `"tf"`, "pt"` or `"jax"` are passed in as the images can't be batched together. 

My preference would be to keep `return_tensors=None` as this more closely matches the behaviour of our tokenizers. However, our `tokenizer` provides arguments such that batches can be created e.g. `padding=True`. Not sure if an equivalent makes sense here. 

What do you think? If we want to set "np" as default we should discuss how to handle introducing the image processors versus introducing that change. 
cc @NielsRogge